crates/types/src/v1alpha1/wasmcloud_host_config.rs

```diff
@@ -101,6 +103,45 @@ pub struct KubernetesSchedulingOptions {
     pub pod_template_additions: Option<PodSpec>,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct ObservabilityConfiguration {
+    #[serde(default)]
+    pub enable: bool,
```

Comment:
Should we be adding the documentation to each of these options so people know what they do? I know we have the command line flags, but wasn't sure if we were documenting these or not

Reply:
We should, I'll follow-up with a PR that does the documenting part, including the README